ocdaction/ocdaction/templates/challenge/challenge_list.html

```diff
@@ -1,48 +1,107 @@
-                <h4>In progress</h4>
-                <li class="row">
-                    Level: {% get_anxiety_level challenge %}
-                    <a href="{% url 'challenge' challenge.id %}"><h3>{{ challenge.challenge_name }}</h3></a>
-                    <a class="btn btn-primary pull-right col-xs-3" href="{% url 'challenge-score-form' challenge.id anxiety_score_card.id %}"> Do the challenge </a>
-                </li>
-                <hr>
-            </ul>
-            {% else %}
-                {% if another_challenge_in_progress %}
-                    <ul class="col-xs-6 col-xs-offset-3">
-                        <li class="row">
-                            Level: {% get_anxiety_level challenge %}
-                            <a href="{% url 'challenge' challenge.id %}"><h3>{{ challenge.challenge_name }}</h3></a>
-                            <button class="btn btn-primary pull-right col-xs-3" disabled> Do the challenge </button>
-                        </li>
-                    </ul>
-                {% else %}
-                    <ul class="col-xs-6 col-xs-offset-3">
-                        <li class="row">
-                            Level: {% get_anxiety_level challenge %}
-                            <a href="{% url 'challenge' challenge.id %}"><h3>{{ challenge.challenge_name }}</h3></a>
-                            <a class="btn btn-primary pull-right col-xs-3" href="{% url 'challenge-score-form-new' challenge.id %}"> Do the challenge </a>
-                        </li>
-                    </ul>
-                {% endif %}
+                <div class="col-lg-12">
+                    <div class="ibox float-e-margins">
+                        <div class="ibox-title">
+                            <h3 class="text-align-center">In progress</h3>
+                        </div>
+                        <div class="ibox-content">
+                            <table class="table table-striped">
+                                <thead>
+                                <tr>
+                                    <th class="ibox-content bold">Level</th>
+                                    <th class="ibox-content bold">Challenge</th>
+                                </tr>
+                                </thead>
+                                <tbody>
+                                    <tr>
+                                        <td class="text-align-left ibox-content">{% get_anxiety_level challenge %}</td>
+                                        <td class="text-align-left ibox-content">
+                                            <a href="{% url 'challenge' challenge.id %}">{{ challenge.challenge_name | title}}</a>
+                                        </td>
+                                        <td class="text-align-right ibox-content"><a class="" href="{% url 'challenge-score-form' challenge.id anxiety_score_card.id %}"><span class="glyphicon glyphicon-hourglass margin-right-5"></span>Continue</a></td>
```

Comment:
no need for 'Continue' here, just an arrow icon will be enough

Reply:
I will hide the text visually, it is still needed for accessibility. Empty links and buttons are not ok.